Upstream-Status: Backport

Without this patch, we'd need to pass --with-sysroot to gn's bootstrap.py to
actually _avoid_ passing --no-sysroot to gen.py, which is confusing.

Signed-off-by: Raphael Kubo da Costa <raphae.kubo.da.costa@intel.com>
---
From 4849d9a19f700961e2c25b642bdaaa6514040e60 Mon Sep 17 00:00:00 2001
From: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>
Date: Tue, 13 Nov 2018 15:41:11 +0000
Subject: [PATCH] gn bootstrap: Remove sysroot-related options

https://gn-review.googlesource.com/c/gn/+/3200 removed the --no-sysroot
option from gen.py and moved to only use a sysroot in the GN bots, so the
options here no longer have any effect.

Bug: 904350
Change-Id: Ieddc71a4f8f802c2d050de1c69ca3a9ec788ee88
Reviewed-on: https://chromium-review.googlesource.com/c/1333391
Reviewed-by: Nico Weber <thakis@chromium.org>
Commit-Queue: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>
Cr-Commit-Position: refs/heads/master@{#607596}
---
 tools/gn/bootstrap/bootstrap.py | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/tools/gn/bootstrap/bootstrap.py b/tools/gn/bootstrap/bootstrap.py
index 261fddde7801..1945852c419d 100755
--- a/tools/gn/bootstrap/bootstrap.py
+++ b/tools/gn/bootstrap/bootstrap.py
@@ -46,10 +46,6 @@ def main(argv):
       '--build-path',
       help='The directory in which to build gn, '
       'relative to the src directory. (eg. out/Release)')
-  parser.add_option(
-      '--with-sysroot',
-      action='store_true',
-      help='Download and build with the Debian sysroot.')
   parser.add_option('-v', '--verbose', help='ignored')
   parser.add_option(
       '--skip-generate-buildfiles',
@@ -76,8 +72,6 @@ def main(argv):
       '--no-last-commit-position',
       '--out-path=' + gn_build_dir,
   ]
-  if not options.with_sysroot:
-    cmd.append('--no-sysroot')
   if options.debug:
     cmd.append('--debug')
   subprocess.check_call(cmd)
-- 
2.19.1

